tests/sles4sap/netweaver_cluster.pm

```diff
@@ -14,7 +14,7 @@ use base "sles4sap";
 use testapi;
 use lockapi;
 use hacluster;
-use utils 'systemctl';
+use utils;
```

Comment:
Why? Not useful to import all `utils` for only one or two functions.

Reply:
The PR is still WIP.  Still missing the takeover part.